protractor-docker/docker.md

```diff
@@ -0,0 +1,147 @@
+Running Protractor Tests on  Docker
```

Comment:
This file is really awesome and should be available to read when the user navigates to this folder on github. This should be changed to README.md.